core/tests/webdriverio_utils/ContributorDashboardAdminPage.js

```diff
@@ -225,9 +241,18 @@ var ContributorDashboardAdminPage = function() {
   };
 
   this.expectStatsElementCountToBe = async function(elementsCount) {
-    await waitFor.visibilityOf(statsTable, 'stats table is not visible');
-    var statsListItems = await statsListItemsSelector();
-    expect(statsListItems.length).toBe(elementsCount);
+    if (elementsCount === 0) {
```

Comment:
Instead of adding logic to this block create two functions `expectNoStatsElement`, `expectStatsElementCountToBe`

Reply:
Done,  PTAL.